### PR TITLE
Style tab bar, nav bar, and status bar.

### PR DESCRIPTION
**Pull request by Lateworm**

### CODE REVIEW

File: js/navigation/NavigationLayout.js [hunk 1]
```diff
@@ -20,9 +23,13 @@ class NavigationLayout extends Component {
         id="main"
         navigatorUID="main"
         initialTab="devRoutes"
-        tabBarColor="#ccc"
+        tabBarColor={nearBlack}
       >
-        <TabItem id="devRoutes" title="Dev Routes">
+        <TabItem
+          id="devRoutes"
+          // title="Dev Routes"
```
Comment (siddharthparmar7) [inline js/navigation/NavigationLayout.js]: Remove this comment, please.

File: js/navigation/NavigationLayout.js [hunk 2]
```diff
@@ -31,7 +38,11 @@ class NavigationLayout extends Component {
           />
         </TabItem>
 
-        <TabItem id="userProfile" title="User Profile">
+        <TabItem
+          id="userProfile"
+          // title="User Profile"
```
Comment (siddharthparmar7) [inline js/navigation/NavigationLayout.js]: Please remove this comment.

File: js/navigation/NavigationLayout.js [hunk 3]
```diff
@@ -40,7 +51,11 @@ class NavigationLayout extends Component {
           />
         </TabItem>
 
-        <TabItem id="companies" title="Companies">
+        <TabItem
+          id="companies"
+          // title="Companies"
```
Comment (siddharthparmar7) [inline js/navigation/NavigationLayout.js]: Please remove this comment.

File: js/navigation/NavigationLayout.js [hunk 4]
```diff
@@ -49,7 +64,13 @@ class NavigationLayout extends Component {
           />
         </TabItem>
 
-        <TabItem id="myProjects" title="My Projects">
+        <TabItem
+          id="myProjects"
+          // title="My Projects"
```
Comment (siddharthparmar7) [inline js/navigation/NavigationLayout.js]: Please remove this comment.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Thank you!

**Comment by Lateworm** [comment]
Commented title code has been removed.

**Review by siddharthparmar7 — APPROVED** [review]
Thank you!

**Pull request merged**